### DISCUSSION TITLE
Where is settings cfg?

### DISCUSSION THREAD

**Discussion by Wladekson** [Moonvy/figmaEX#13] [Q&A]
I need to reinstall windows, and therefore figma. Can I back up FigmaEx settings (labels and labels order)?

**Comment by yArna** [comment]
You can open the settings and click on the `Export Config` button.
The config will be exported as a text layer to the currently open figma file.

You can import the config through this layer again, using the `Import Conig` button.

**Reply by Wladekson** [reply]
Thanks, didn't notice it

**Comment by ferkat0** [comment]
Hey! How do I make the panel horizontal?

**Reply by yArna** [reply]
Not supported yet, but it might be a good idea

**Comment by ferkat0** [comment]
It will be very good idea)

I miss click on random plugin when I wanna use right panel in figma

сб, 22 мая 2021 г., 13:26 yArna ***@***.***>:
